transplant_guide.md

```diff
@@ -120,29 +124,38 @@ Known as the most LGBT friendly neighborhood. Tons of cool shops and restaurants
 
 **First Hill**  
-Info about First Hill
+Used to also be referred to as pill hill, because there are a lot of hospitals in the area. If you are easily awoken by ambulance sirens, it may not be the best place to live. The location is very convenient as it is close to downtown and to Capitol Hill, but there is not often a ton of open housing in the area. There are a few new complexes under construction in the area but will likely be pretty pricey when they open, and there is at least one microhousing building in the neighborhood.
 
 **Fremont**  
-Info about Fremont
+Fremont is the self-coined Center of the Universe, and can be a fun place to live if you can find housing-- it often is sparse and expensive. It is the location of the solstice parade (which includes a naked bike ride) and is home to many bars, restaurants, breweries and tap rooms. It is north of queen anne from downtown, or west of Wallingford on the north side of lake union. From downtown, there is a bridge that is low enough it has to open for boat traffic, so that can cause some traffic problems from time to time. It is on the Burke Gilman trail (fun bike ride) and it not too terrible of a bike commute to downtown(but again, hills, because Seattle).
 
 **Ballard**  
-Info about Ballard
+Ballard is kind of new hipster central in Seattle, with new craft cocktail gastropubs and microbreweries seemingly popping up weekly. It is a pretty fun place to go out on the weekends and they have an awesome Sunday farmer's market. There is quite a bit of new construction, but it is pretty pricy given it is brand new.
+Getting downtown via transit isn't impossible, but because it is further west, it can make getting to sort of inconvenient at times.  It would not be convenient to commute to an east side (aka Bellevue (Expedia/Concur) or Redmond (Microsoft)) internship or job, but if you are coming from a place where you have an hour+ commute, maybe it wouldn't be that noticeable to you.  
 
 **University District (U-District)**
-Info about U-district
+This is where the University of Washington is located, so there are a lot of rental properties in this area. There are a lot of students in this neighborhood which can be both good and bad- kind of depends on how you feel about large numbers of 18-22 year olds. There are a lot of cheap, fast places to eat and coffee shops that are open late.
+It is a good place to look for sublets, but realize that your housemates may be college students and it is hard (aka stressful) to move during Ada.
+It has fairly accessible transit to downtown, but it is a bit further away than other neighborhoods and is a somewhat hilly bike commute. Eventually it will be connected to downtown via light rail, but that won't happen until 2021, so probably not a strong consideration at this point.
```

Comment:
This is wrong. The light rail will connect through cap hill to the montlake side of the u district in 2016. In 2021, the first half of the lynnwood extension (ie, the northgate extension) will open, which will include a stop in the west side of the u district and another in ravenna. In 2023, east link & the rest of the lynnwood line should be open.

Sound Transit: http://www.soundtransit.org/Projects-and-Plans/Find-a-Project